src/api/app/models/user.rb

```diff
@@ -911,6 +911,14 @@ def update_notifications(params)
     User.update_notifications(params, self)
   end
 
+  def combined_rss_feed_items
+    feed_items = rss_feed_items.to_a
```

Comment:
It would be much nicer if this could return an `ActiveRecord::Relation` instead of an `Array`.  Could you do something like this? 

```rb
Notification::RssFeedItem.where(user: self).or(
  Notification::RssFeedItem.where(group: groups)
)
```
This might work too:
```rb
rss_feed_items.or(
  Notification::RssFeedItem.where(group: groups)
)
```